poly-markdown.el

```diff
@@ -72,6 +73,12 @@
   :mode-matcher (cons "`[ \t]*{?\\(?:lang *= *\\)?\\([[:alpha:]+-]+\\)" 1)
   :allow-nested nil)
 
+(defun poly-markdown-fenced-code-begin-accessor (val)
+  (cons (point)
```

Comment:
Why not just `(cons (car val) (cadr val))`? Those are the locations of the property.

Reply:
In fact I would prefer to match without relying on the accessor because those properties could be potentially moved and not immediately updated.  

This increment at the end of the property is a common requirement. YAML should also do that. So I have implemented a simple :inc-end accessor for pm-make-text-property-matcher.